src/sys/mod.rs

```diff
@@ -18,11 +18,15 @@ use libc::*;
 #[cfg(all(
     not(feature = "bindgen"),
     not(any(
-        target_arch = "x86_64",
         target_arch = "aarch64",
-        target_arch = "riscv64",
+        target_arch = "arm",
         target_arch = "loongarch64",
-        target_arch = "powerpc64"
+        target_arch = "powerpc",
```

Comment:
I'm a bit hesitant about vendor more archs. This should be a separate PR.

Reply:
I can move these to a separate PR.

Is there a reason to prefer not to include these while some other Tier 2 targets are already in place?

The current behavior of falling back to the `x86_64` sys module seems worse than including these given the potential for ABI issues.

Reply:
I actually suggestion that tier 2 users specify their own sys file. that's what I do myself.

In the long term I hope to migrate to `linux-raw-sys`, but at the same time keep the `bindgen` feature.